src/backend/utils.ts

```diff
@@ -1255,6 +1256,10 @@ export function getExecutableAndArgs(executableWithArgs: string): {
   const executable = match ? match[0] : ''
   const launchArgs = executableWithArgs.replace(executable, '').trim()
 
+  if (isAbsolutePath) {
+    return { executable: executableWithArgs, launchArgs }
+  }
+
```

Comment:
having trouble understanding this one. so if the exe path is an absolute path, then we don't want to return the exe with the launch args stripped?

What happens when a exe with args i.e `"/usr/123/asdf/aaa--arg=1"` is passed to `isAbsolute`? Does it return false or does it throw? Seems that might be an unhandled edge case.

Reply:
If you add some console.logs you will be able to see that the `exeOnly` is coming empty and then it first fails to check permissions and then fails to chmod it and finally fails to launch it.

Reply:
sorry I meant to write `"/usr/123/asdf/aaa --arg=1"` with the space. Won't that come back as `isAbsolutePath` false and then return `{ executable: "/usr/123/asdf/aaa --arg=1"...` ?

Reply:
Also could we add tests for this function since it is a pure utils function?

Reply:
I see, yes, that was one good edge case. should be handled now.